core/src/main/java/org/apache/calcite/rel/logical/LogicalCalc.java

```diff
@@ -117,9 +115,7 @@ public static LogicalCalc create(final RelNode input,
     final RelTraitSet traitSet = cluster.traitSet()
         .replace(Convention.NONE)
         .replaceIfs(RelCollationTraitDef.INSTANCE,
-            () -> RelMdCollation.calc(mq, input, program))
-        .replaceIf(RelDistributionTraitDef.INSTANCE,
-            () -> RelMdDistribution.calc(mq, input, program));
+            () -> RelMdCollation.calc(mq, input, program));
```

Comment:
These have been here for 10 years. @julianhyde seems to be the one who has added them, maybe he can say whether this change makes sense.

Reply:
Yes, maybe Julian can help me with that.